sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/OfferUtilsTest.java

```diff
@@ -1,8 +1,10 @@
 package com.mesosphere.sdk.offer;
```

Comment:
I may just not be seeing where you test the alien case of isProcessable or clean in OfferUtils?

Reply:
https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/ResourceUtilsTest.java#L18-L120